sdcm/nemesis.py

```diff
@@ -2223,6 +2243,16 @@ def disrupt_delete_by_rows_range(self):
 
         self.delete_range_in_few_partitions(ks_cf, partitions_for_exclude)
 
+    def disrupt_delete_with_timestamp(self):
+        """
+        Delete a large amount of rows from a single partition using WITH TIMESTAMP clause
+        """
+        self.verify_initial_inputs_for_delete_nemesis()
+
+        ks_cf = 'scylla_bench.test'
```

Comment:
so, it works only when scylla-bench is used, add this info to docstring, also make nemesis skip if no such ks_cf.

Reply:
I don't see technical dependency on the scylla-bench here.